### PR TITLE
AB#34250: Removed HTAStatus Entity

### PR DESCRIPTION
**Pull request by Flemingjp**
Removes HTAStatus Entity and all references to it. In preparation for the Aggregator

### CODE REVIEW

**Review by JosephLavagna — APPROVED** [review]

**Review by blargly — APPROVED** [review]

**Comment by Flemingjp** [comment]
Awaiting #206 + Migration

File: elastic-search/directory index setup/collections.json [hunk 1]
```diff
@@ -63,9 +63,6 @@
           }
         }
       },
-      "hta": {
```
Comment (beforan) [inline elastic-search/directory index setup/collections.json:66]: Can we add a labeler config to label `Contains: Search Index` or something, to indicate index structure changes on this PR?

It should be for the path in `./elastic-search/directory index setup/*` I think?

**Review by beforan — CHANGES_REQUESTED** [review]

**Review by JosephLavagna — DISMISSED** [review]

**Review by beforan — APPROVED** [review]

**Review by beforan — CHANGES_REQUESTED** [review]
labeler won't work how we think

File: .github/workflows/label.yml [hunk 2]
```diff
@@ -6,7 +6,9 @@
 # https://github.com/actions/labeler
 
 name: Labeler
-on: [pull_request]
+on:
+  pull_request:
+    types: [ready_for_review]
```
Comment (beforan) [inline .github/workflows/label.yml:11]: this trigger isn't quire what we want. instead we want a condition on the job itself, so it doesn't run for `draft` PRs, even though the workflow always triggers on normal PR events

**Review by beforan — CHANGES_REQUESTED** [review]

**Review by beforan — CHANGES_REQUESTED** [review]

**Review by beforan — APPROVED** [review]

**Review by JosephLavagna — APPROVED** [review]

**Pull request merged**